content/guides/exporters/supported-exporters/Go/Honeycomb.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Honeycomb.io (Tracing)"
+date: 2018-09-19
+draft: false
+weight: 3
+class: "resized-logo"
+aliases: [/supported-exporters/go/honeycomb]
+logo: /img/honeycomb-logo.jpg
+---
+
+- [Introduction](#introduction)
+- [Creating the exporter](#creating-the-exporter)
+- [Viewing your traces](#viewing-your-traces)
+- [Project link](#project-link)
+
+## Introduction
+
+Honeycomb is a saas tool that provides real-time system debugging. Visualize individual traces to deeply understand request execution. Break down, filter, and aggregate trace data to uncover patterns, find outliers, and understand historical trends.
+
+Honeycomb aggregates data at read time, so you don’t have to predict ahead of time which metrics matter. You can fluidly go between time-series graphs, traces, and raw rows to get answers, without having to switch context between tools.
```

Comment:
Let's add a line or two here that tries to differentiate us from the other tracing exporters:

Honeycomb Tracing offers an unparalleled ability to run high-level queries in order to identify **which** traces are worth investigating, before digging deep into the details of an individual exemplar trace. Easily go from insights on your application from a bird's eye view, to a classic waterfall view of a specific trace, and back. For a walkthrough of this flow in action, [visit our blog](https://www.honeycomb.io/blog/2018/07/there-and-back-again-a-honeycomb-tracing-story/).